config/default.schema.json

```diff
@@ -113,6 +113,11 @@
 			"type": "boolean",
 			"default": false,
 			"description": "If enabled, user will see messages/incidents from e.g status.schul-cloud"
+		},
+		"N21_BLOG_CONTENT_API_KEY": {
+			"type": "string",
+			"default": "9cbea173e6e7ecb709e8f9c960",
```

Comment:
NEVER EVER SET DEFAULT KEYS, AS THEY ARE [SECRET] KEYS!
Instead: use a feature flag which defaults to false, in code on startup throw, if the feature flag is true and the api key is not valid/missing/whatever...

Reply:
locally, add the secret into a .env

Reply:
content key is not secret and can easily be accessed by just looking into the browser inspector